ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -820,6 +820,8 @@ then
 			fmu mode_pwm4
 		fi
 
+		pwm failsafe -c 1234 -p 900
```

Comment:
This is hardcoding the PWM value which should be read from the PWM_DISARMED parameter.